src/utils/update/psc.to.beneficial.owner.type.mapper.ts

```diff
@@ -0,0 +1,170 @@
+import { OfficeAddress } from "@companieshouse/api-sdk-node/dist/services/company-profile/types";
+import { CompanyPersonWithSignificantControlResource } from "@companieshouse/api-sdk-node/dist/services/company-psc/types";
+import { BeneficialOwnerGov } from "../../model/beneficial.owner.gov.model";
+import { BeneficialOwnerIndividual } from "../../model/beneficial.owner.individual.model";
+import { BeneficialOwnerOther } from "../../model/beneficial.owner.other.model";
+import { Address, InputDate, NatureOfControlType, yesNoResponse } from "../../model/data.types.model";
+
+export const mapPscToBeneficialOwnerTypeIndividual = (psc: CompanyPersonWithSignificantControlResource): BeneficialOwnerIndividual => {
+  const service_address = mapAddress(psc.address);
+  const result: BeneficialOwnerIndividual = {
+    id: psc.links?.self,
+    first_name: psc.name_elements?.forename,
+    last_name: psc.name_elements?.surname,
+    nationality: psc.nationality,
+    // second_nationality: undefined,
+    date_of_birth: mapDateOfBirth(psc),
+    is_service_address_same_as_usual_residential_address: isSameAddress(service_address) ? yesNoResponse.Yes : yesNoResponse.No,
```

Comment:
isSameAddress should have two args

Reply:
One arg parsed because usualresidential is undefined and was removed for now. 
Parsing undefined as second arg

Reply:
fixed